modules/javafx.graphics/src/main/java/javafx/css/TransitionEvent.java

```diff
@@ -94,14 +100,17 @@ public final class TransitionEvent extends Event {
      *
      * @param eventType the event type
      * @param property the {@code StyleableProperty} that is targeted by the transition
+     * @param propertyName the name of the targeted CSS property or sub-property
      * @param elapsedTime the time that has elapsed since the transition has entered its active period
      * @throws NullPointerException if {@code eventType}, {@code property} or {@code elapsedTime} is {@code null}
      */
     public TransitionEvent(EventType<? extends Event> eventType,
                            StyleableProperty<?> property,
+                           String propertyName,
```

Comment:
Sure, but I don't really mind:
1. `TransitionEvent` is a CSS event, it can not usefully be created by application developers.
2. This API (without `propertyName`) will be introduced with JFX 23, so developers will need to hurry if they intend to misuse it until JFX 24 rolls around...

I wouldn't mind removing the public constructor altogether, but this is not something that we usually do for JavaFX events.